docker/build-tools/Dockerfile

```diff
@@ -96,16 +96,18 @@ FROM golang:1.12 as build_hugo_context
 ENV HUGO_VERSION=0.55.5
 RUN mkdir -p /out/go/bin
 ADD https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz /tmp
-RUN tar -xzvf /tmp/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz -C /tmp
-RUN mv /tmp/hugo /out/go/bin
+RUN tar -xzvf /tmp/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz --strip-components=1 -C /usr/local
 
 # Build a nodejs context
-FROM ubuntu:xenial as build_nodejs_context
-RUN apt-get update
-RUN apt-get -y install curl
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
-RUN apt-get -y install nodejs
+FROM ubuntu:xenial as node_tools_context
```

Comment:
Update comment to be node tools instead of nodejs

Reply:
addressed.